main.js

```diff
@@ -2,5 +2,127 @@
  * Created by Jenya on 18.04.15.
  */
 
+/// At first I'm creating general BeingClass. It will be the top of all being classes.
+function BeingClass(clas) {
+    this.clas = clas;
```

Comment:
what does `clas` mean here? it is confusing, I thought you were going to use name